engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -2092,6 +2068,35 @@ public void populateDAG(LogicalPlan dag)
 
   }
 
+  private void populateOperators(AppConf appConf, LogicalPlan dag, Map<OperatorConf, Operator> nodeMap)
```

Comment:
Thanks, there is no rule that says a method can only have 100 lines. If there is a good case for refactor, then do it as separate change, don't mix it with an already giant and difficult to review pull request.